ui/raidboss/data/04-sb/trial/seiryu-ex.js

```diff
@@ -212,20 +212,7 @@
       regexJa: Regexes.ability({ id: '37E5', source: '山の式鬼', capture: false }),
       regexCn: Regexes.ability({ id: '37E5', source: '山之式鬼', capture: false }),
       regexKo: Regexes.ability({ id: '37E5', source: '산 사역귀', capture: false }),
-      infoText: {
-        en: 'East =>',
-        de: 'Osten =>',
-        fr: 'Est =>',
-        ja: '東 =>',
-        cn: '东 =>',
-      },
-      tts: {
-        en: 'East',
-        de: 'Osten',
-        fr: 'Est',
-        ja: 'ひがし',
-        cn: '东 =>',
-      },
+      response: Responses.goEast(),
```

Comment:
This is another go east / right => situation.  Maybe we just need responses for that.

Reply:
goEast() will equal to `en: 'Get Right/East =>',` so i think this is fine